keras/src/losses/losses.py

```diff
@@ -1961,28 +1961,42 @@ class Dice(LossFunctionWrapper):
 
     Returns:
         Dice loss value.
+
+    Examples:
+
+    y_true = tf.constant([[[[1.0], [1.0]], [[0.0], [0.0]]], [[[1.0], [1.0]], [[0.0], [0.0]]]])
```

Comment:
Please use a code block for the code example. Do not use TF APIs, only Keras APIs (backend agnostic)

Reply:
Updated